scripts/check-collections

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# This script checks if collection types in TiKV.
+set -euo pipefail
+
+function error_msg() {
+    echo "Prefer collections::HashMap" >&2
+}
+
+rules=(
+    "\{collections::.*HashMap[^<:E]"
```

Comment:
Use [disallowed_types](https://rust-lang.github.io/rust-clippy/master/index.html#disallowed_types) in clippy.toml?